proto/process/agent.proto

```diff
@@ -651,11 +651,14 @@ enum EphemeralPortState {
 	ephemeralFalse = 2;
 }
 
+message ProtocolStack {
+	repeated ProtocolType stack = 1;
+}
```

Comment:
Is the expectation that this is always going to be spare? e.g. 

stack[0] = mainProtocol (even if empty)
stack[1] = l7 protocol (even if empty)
stack[2] = l4 security


Without knowing that, we'll need to have a hardcoded mapping of protocolHTTp/protocolHTTP2/etc.  in the backend, else drop these tags. I suppose that's not the end of the world.

Reply:
the right most protocol should be the most "precise" protocol
it will be "tcp", "tls", "http", "aws sqs" for example

I can split it back to the format you've suggested, but @nplanel pointed out that `tcp` or `udp` is being reported by the `type` field of the connection
and then we will have just 1 protocol for "security", so it seemed redundant